src/components/Layout/TheFooter.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div id="footer">
+    <span> {{ `&copy; ${year}` }} | Built with love by <a target="_blank" href="https://github.com/orgs/CodingGardenCommunity/people">CodingGarden Community</a> </span>
```

Comment:
Wouldn't it make more sense to link directly to the contributors page? `/contributors`

Reply:
Ah. That's a good idea. Thanks!

Reply:
Updated. Added `router-link` to `/contributors`.